code/espurna/migrate.ino

```diff
@@ -964,6 +964,16 @@ void migrate() {
             setSetting("relayGPIO", 0, 5);
             setSetting("relayType", 0, RELAY_TYPE_NORMAL);
 
+        #elif defined(AVATTO_NAS_WR01W)
+
+            setSetting("board", 75);
+            setSetting("ledGPIO", 0, 13);
+            setSetting("ledLogic", 0, 1);
+            setSetting("btnGPIO", 0, 0);
+            setSetting("btnRelay", 0, 0);
+            setSetting("relayGPIO", 0, 14);
+            setSetting("relayType", 0, RELAY_TYPE_NORMAL);
```

Comment:
Do not trust the values in this file. (See PR comment)

Reply:
migrate() setup code effectively does nothing atm, but thanks for noticing.

I will be removing this whole part of the function though, so technically anything here that does not break build is ok.